### PR TITLE
node@15 + next@10

### PR DESCRIPTION
**Pull request by Drapegnik**
```bash
nvm use # for set node version
```

- node@15
- next@10
- remove webpack plugins
- remove storybook
- remove bundle-analyzer

### CODE REVIEW

File: .github/workflows/lint-build.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: lint & build
+
+on: push
+
+jobs:
+  lint-build:
+    runs-on: ubuntu-latest
+    env:
+      BABAJKA_GOOGLE_API_KEY: ${{ secrets.BABAJKA_GOOGLE_API_KEY }}
```
Comment (UladBohdan) [inline .github/workflows/lint-build.yml:9]: btw why do we need in CI ? to fetch data on postinstall? do we really want to fetch it here?

File: bin/deploy/install-deps.sh [hunk 2]
```diff
@@ -3,4 +3,5 @@ set -e
 
 cd "/home/wir-$1/deployed/swap-frontend/babajka-frontend"
 
-npm ci --ignore-scripts
+npm ci --ignore-scripts || true # HACK to ignore failing postinstall script.
```
Comment (UladBohdan) [inline bin/deploy/install-deps.sh:6]: What are we going to do with that?..

File: package.json [hunk 3]
```diff
@@ -2,10 +2,14 @@
...
   "description": "Next.js & React app",
+  "engines": {
+    "node": "~15",
+    "npm": "~7.1.2"
+  },
   "scripts": {
     "start": "npm run dev",
     "start:prod": "npm run build && cross-env NODE_ENV=production node server.js",
-    "start:prod:remote": "node server.js",
+    "start:prod:remote": "node -v && node server.js",
```
Comment (UladBohdan) [inline package.json]: Remove `node -v` ?

Comment (UladBohdan) [reply package.json]: If we want to print remote node version, we can put `node -v` to bash script explicitly

File: pages/_app.js [hunk 4]
```diff
@@ -85,15 +85,15 @@ class Root extends App {
...
       loadYM(YM_ID);
       window.ym(YM_ID, 'hit', url);
     }
-    if (__PROD__ && !window.ga) {
+    if (process.env.isProd && !window.ga) {
       ReactGA.initialize(GA_ID, { debug: false });
       ReactGA.ga('send', 'pageview', url, { hitCallback: clearUtmParams });
     }
-    if (!__PROD__) {
+    if (!process.env.isProd) {
```
Comment (UladBohdan) [inline pages/_app.js:96]: why don't we clean utm params on prod?

File: bin/deploy/install-deps.sh (hunk 2 above)
Comment (Drapegnik) [reply bin/deploy/install-deps.sh:6]: I think it's ok)

File: pages/_app.js (hunk 4 above)
Comment (Drapegnik) [reply pages/_app.js:96]: on prod they cleaned by google-analytics hook:
```
ReactGA.ga('send', 'pageview', url, { hitCallback: clearUtmParams });
```

**Pull request merged**